chapters/Chapter 1 — Untitled.md

```diff
@@ -49,14 +49,76 @@ Something wet was pooling at the bottom of her helmet’s goggles. Right: she wa
+
+Leanne forced herself to watch until the end. Her aim hadn’t been perfect. She’d hit the woman’s neck instead of her head.
+
+The woman’s twitching slowed. The fury in her boiling eyes dimmed. 
+
+Leanne took a deep breath, and crawled into her attacker’s ship. Her head spun as she landed in the remains of the cockpit. 
+
+She checked the feeds from the cameras. It appeared she was alone, or alone enough. She rifled through the console. Her attackers, like her, were assassins. They should have a card for the job. It would have all the pertinent details, and they’d turn it in with proof of death. 
+
+It was unlikely there’d be anything on the card to trace back to the submitter; assassins always worked through an Arranger, and even the Arranger rarely knew who hired them. But Leanne searched all the same, whether out of habit or hunch.
+
+The card was small. There was a little yellow insignia on the upper-left of a diving eagle—the Leesome Arrangers, if Leanne remembered correctly. There was little information on it, besides a picture of her and her ship, and a location she once frequented years ago. It was extremely impressive that they found her at all. 
+
+There was another card. Same insignia. It was odd: Arrangers usually only handed out one card at a time to any given assassin. Leanne always assumed it was because the cards themselves were expensive due to their cryptographic something or others. 
+
+On the second card was a woman. She lived on Dressara. Short black hair, angular face, and according to the card, not all that dangerous.
+
+Could be important. Perhaps there was a link between this woman and Leanne. Or perhaps she was just another job, handed out at the same time. There were two attackers, after all; perhaps the Leesomes had given them each their own job.
+
+Leanne pocketed the card. She looked out the window. Her ship was a speck in the distance. She glanced around the remains of the cockpit. It looked like it could still be flyable. 
+
+She flicked the switches. Pulled the handles. The ship’s many engines weren’t meant to be controlled by a human, but she was managing to bring it around rather gracefully.
+
+In the distance, she could see her ship, still spinning. Inside was probably a mess by now. She’d clean it up eventually.
+
+The distance closed. She shook her head as she accelerated, and did precisely what her attackers had done before: she rammed her own ship.
+
+The force broke her ship from its spin. It flew away, still spinning, if lightly.
+
+Leanne jumped. She swung her legs up, as if doing a backflip, and aimed herself at her ship. 
+
+She should have fallen on her head, but instead, gravity rearranged itself around her. It pulled her out of the damaged mashed-together ship, through the nothingness, home.
```

Comment:
Leanne didn’t remember entering through the hold, closing the airlock doors behind her, and collapsing on her bed. It was alright. She was used to not remembering.